templates/cube/microcerts.html

```diff
@@ -93,11 +97,11 @@ <h2>Badge of honor</h2>
             loading="auto"
             ) | safe
           }}
-          </a>
+        </a>
         </li>
       </ul>
-      <p>You can <a href="https://www.linkedin.com/profile/add?startTask=CERTIFICATION_NAME&name=Test%20Certificate&organizationName=LinkedIn&issueYear=2018&issueMonth=2&expirationYear=2020&expirationMonth=5&certUrl=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Flearn%2Fcertifications%2Fd365-functional-consultant-sales&certId=1234">add this badge</a> to the Certification section of your LinkedIn profile</p>
-      <p class="u-no-padding--top"><small>https://api.badgr.io/public/assertions/MPhZ1kQGT061-hl</small></p>
+      <p>You can <a href="https://www.linkedin.com/profile/add?startTask=CERTIFICATION_NAME&name=CUBE 2020&organizationName=Canonical&certUrl={{ certified_badge.share_url }}">add this badge</a> to the Certification section of your LinkedIn profile</p>
+      <p class="u-no-padding--top"><small>{{ certified_badge.share_url }}</small></p>
```

Comment:
I think here we just want to share `certified_badge.image` as discussed in standup. I will probably remove `share_url`.

Reply:
ok will do, thanks